### DISCUSSION TITLE
Windows desktop: how to trigger on purpose the fallback to username/password

### DISCUSSION THREAD

**Discussion by gramunno** [gravitational/teleport#47333] [Q&A] [answered]
Hello,
in order to have two levels of authentication (Teleport + MS Windows) how is it possible to force the fallback to username/password for Windows desktop access?

**Comment by zmb3** [comment, marked as answer]
It is not possible. Teleport does passwordless-only authentication to Windows hosts.

**Reply by gramunno** [reply]
Thanks!
